data/mods/Magiclysm/Spells/attunements/Cleansing_Flame.json

```diff
@@ -75,5 +75,62 @@
     "min_range": 4,
     "max_range": 8,
     "range_increment": 0.12
+  },
+  {
+    "id": "cleansingflame_hermes_heal",
+    "type": "SPELL",
+    "spell_class": "CLEANSING_FLAME",
+    "name": "Hermes reborn",
+    "description": "In the sea without lees \nStandeth the bird of Hermes \nEating his wings variable \nAnd maketh himself yet full stable",
+    "valid_targets": [ "self" ],
+    "effect": "attack",
+    "shape": "blast",
+    "base_casting_time": 350,
+    "base_energy_cost": 800,
+    "energy_source": "MANA",
+    "difficulty": 10,
+    "min_damage": -20,
+    "max_damage": -120,
+    "damage_increment": -2.5,
+    "max_level": 40,
+    "extra_effects": [ { "id": "cleansingflame_hermes_burn" }, { "id": "cleansingflame_hermes_split" } ],
+    "flags": [ "SOMATIC", "VERBAL", "CONCENTRATE", "NO_PROJECTILE", "IGNORE_WALLS", "LOUD", "MUST_HAVE_CLASS_TO_LEARN" ]
+  },
+  {
+    "id": "cleansingflame_hermes_burn",
+    "type": "SPELL",
+    "name": "Hermes reborn fire",
+    "description": "Give some frost damage for hermes reborn spell. Technical spell, you dont allowed to see it, if youre not in debug mode.",
+    "valid_targets": [ "self", "ally", "hostile", "ground" ],
+    "flags": [ "CONCENTRATE", "SOMATIC", "LOUD", "NO_PROJECTILE" ],
+    "effect": "attack",
+    "shape": "blast",
+    "min_damage": 10,
+    "max_damage": 25,
+    "damage_increment": 0.4,
+    "min_range": 1,
+    "max_range": 1,
+    "damage_type": "cold",
+    "min_aoe": 3,
+    "max_aoe": 5,
+    "aoe_increment": 0.05,
+    "spell_class": "CLEANSING_FLAME",
+    "difficulty": 4,
+    "max_level": 40,
+    "energy_source": "MANA"
+  },
+  {
+    "id": "cleansingflame_hermes_split",
+    "type": "SPELL",
+    "name": "Hermes reborn pain split",
+    "description": "Same as biomants pain sprit. Technical spell, you dont allowed to see it, if youre not in debug mode",
+    "valid_targets": [ "self" ],
+    "effect": "pain_split",
+    "shape": "blast",
+    "energy_source": "MANA",
+    "flags": [ "SOMATIC", "NO_LEGS", "CONCENTRATE" ],
+    "spell_class": "CLEANSING_FLAME",
+    "difficulty": 2,
+    "max_level": 1
```

Comment:
```suggestion
```

Reply:
I thought i copied already uptaded version. Sorry for that and thanks

Reply:
Understand, and quite welcome! You'll need to click a button labeled something like "accept suggestions" (possibly as a batch - check if there's an arrow pointing down enabling more than one selection) to do that.